src/UI/Implementation/Component/Button/Button.php

```diff
@@ -99,13 +99,13 @@ public function appendOnClick(Signal $signal) {
 	 * @inheritdoc
 	 */
 	public function withOnHover(Signal $signal) {
-		return $this->addTriggeredSignal($signal, 'hover');
+		return $this->addTriggeredSignal($signal, 'mouseenter');
```

Comment:
If there is only one callback function provided when binding handlers for the `hover` event in JQuery, it calls the function for both `mouseenter` and `mouseleave` events. Which lead to double triggered signals.